### PR TITLE
Assume Identity for Admins

### PR DESCRIPTION
**Pull request by swards**
## Description
This allows Admins to login as another user and see what they see.
Routes updated, must be an Admin

## Screen Shots
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Ruby on Rails 7 running with Postgres DB
 ## Downloading and Installing
 Clone repo from git and start the Rails app as usual.
 - Bundle: `bundle`
-- Build the database: `rails db:create`
+- Build the database: `rails db:schema:load`
```
Comment (swards) [inline README.md:12]: This is a separate issue - I saw this in the readme, I think we want to encourage new devs to load from the schema.

Comment (SiobhanCoady) [reply README.md:12]: I'm just getting back into the swing of things here, but it seems to me like someone newly setting up their dev environment would still need to run `rails db:create` first. After that, `rails db:schema:load` works great. Going straight to `rails db:schema:load` left me with this error:
```
$ rails db:schema:load
rails aborted!
ActiveRecord::NoDatabaseError: We could not find your database: looseends_development. Which can be found in the database configuration file located at config/database.yml.

To resolve this issue:

- Did you create the database for this app, or delete it? You may need to create your database.
- Has the database name changed? Check your database.yml config has the correct database name.

To create your database, run:

        bin/rails db:create
```

**Review by SiobhanCoady — APPROVED** [review]
One question/comment about the readme change, but the rest looks great!

File: README.md (hunk 1 above)
Comment (swards) [reply README.md]: Thanks Siobhan!

**Pull request merged**